framework/kvstore/kvstore_test.go

```
@@ -98,3 +98,104 @@ func TestStoreClose(t *testing.T) {
+	go func() {
+		defer close(done)
+		<-start
+		for i := 0; i < 1000; i++ {
+			store.SetDelegate(testSyncDelegate{})
+			store.SetDelegate(nil)
+		}
+	}()
+	close(start)
+	for i := 0; i < 1000; i++ {
+		store.SupportsProcessLocalCAS()
+		store.CompareAndSwapStringWithTTL("binding", "a", "a", time.Hour)
```

golangci-lint (2.13.2)

[error] Error return value of `store.CompareAndSwapStringWithTTL` is not checked

(errcheck)